pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/barnepensjon/migrering/EnkeltVedtakOmregningNyttRegelverk.kt

```diff
@@ -0,0 +1,177 @@
+                )
+            }
+
+            paragraph {
+                list {
+                    item {
+                        text(
+                            Language.Bokmal to "Barnepensjonen din øker.",
+                            Language.Nynorsk to "",
+                            Language.English to ""
+                        )
+                    }
+                    item {
+                        text(
+                            Language.Bokmal to "Du får barnepensjon til du blir 20 år.",
+                            Language.Nynorsk to "",
+                            Language.English to ""
+                        )
+                    }
+                }
+            }
+
+            title2 {
+                text(
+                    Language.Bokmal to "Hvordan pensjonen din er beregnet",
+                    Language.Nynorsk to "",
+                    Language.English to ""
+                )
+            }
+            paragraph {
+                textExpr(
+                    Language.Bokmal to "Barnepensjonen din blir lik en ganger folketrygdens grunnbeløp per år. Grunnbeløpet i januar 2024 er ".expr() +
+                            grunnbeloep.format() + " kroner. Dette deles på 12 måneder. Du vil derfor få " +
+                            utbetaltEtterReform.format() + " kroner per måned før skatt fra 1. januar 2024.",
+                    Language.Nynorsk to "".expr(),
+                    Language.English to "".expr()
+                )
+            }
+
+            paragraph {
+                textExpr(
+                    Language.Bokmal to "For å få full pensjon må avdødes trygdetid være beregnet til minst 40 år. Trygdetiden tilsvarer det antall år avdøde har vært medlem i folketrygden etter fylte 16 år. Når avdøde var under 67 år ved dødsfallet blir det beregnet framtidig trygdetid. Det er vanligvis fram til og med det året avdøde ville ha fylt 66 år. Barnepensjonen din er beregnet med".expr() + anvendtTrygdetid.format() + " år.",
+                    Language.Nynorsk to "".expr(),
+                    Language.English to "".expr()
+                )
+            }
+
+            title2 {
+                text(
+                    Language.Bokmal to "Skattetrekk på barnepensjon",
+                    Language.Nynorsk to "",
+                    Language.English to ""
+                )
+            }
+            paragraph {
+                text(
+                    Language.Bokmal to "Barnepensjon er skattepliktig, men vi trekker ikke skatt uten å få beskjed om dette. Du kan lese mer om skatt på barnepensjon på nav.no/skatt-pengestotte. Her finner du også informasjon om frivillig skattetrekk og hvordan du får registrert dette.",
+                    Language.Nynorsk to "",
+                    Language.English to ""
+                )
+            }
+
+            title2 {
+                text(
+                    Language.Bokmal to "Du har rett til å klage",
```

Comment:
For dei andre breva har vi lagt opp eigne gjenbrukbare komponentar for rett til å klage, spørsmål og dei andre faste seksjonane. Dette brevet er vel kanskje berre tenkt til eingongsbruk, så i så fall er det vel ikkje så farleg, men til neste gong.

Reply:
Jepp, gikk enkleste veien med tanke på at det er til engangsbruk :p